dpcpp/solver/gmres_kernels.dp.cpp

```diff
@@ -252,6 +255,9 @@ void update_next_krylov_kernel(
     size_type stride_krylov, const ValueType* hessenberg_iter,
     size_type stride_hessenberg, const stopping_status* stop_status)
 {
+    if (stride_krylov == 0) {
```

Comment:
Just as a side note from seeing this, should we make sure that is a user provides a stride that it is always >= the number of columns?

Reply:
stride >= num columns is an (implicit) invariant of the matrix format, so that would be caught with a `validate_data` #770